index.html

```diff
@@ -0,0 +1,452 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <script type="text/javascript" src="javascript/main.js" defer></script>
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars">
+        <i class="fas fa-bars"></i>
+      </button>
+      <nav id="desktop-nav">
+        <div class="cancel">&times;</div>
+        <ul>
+          <li class="nav_link">
+            <a class="nav-link" href="#projects">Portoflio</a>
+          </li>
+          <li class="nav_link">
+            <a class="nav-link" href="#about-me">About</a>
+          </li>
+          <li class="nav_link">
+            <a class="nav-link" href="#contact"
+              >Contact <i class="far fa-envelope"></i
+            ></a>
+          </li>
+        </ul>
+      </nav>
```

Comment:
- [ ] It seems you have put much effort into matching the design But Please try to stick to Figma design as much as possible to make your project looks good. As you learn to become a professional developer you have to be able to follow project requirements strictly Because in the future you will need to implement the company's project mockups into real projects. and you are good to go 👍🏾